uap_tracker/dense_optical_flow.py

```diff
@@ -3,10 +3,11 @@
 
 
 class DenseOpticalFlow():
-    def __init__(self, width, height):
+    def __init__(self, width, height, enable_cuda):
         self.previous_frame = None
         self.width = width
         self.height = height
+        self.enable_cuda = enable_cuda
```

Comment:
Not sure what this is for

Reply:
I want to make the optical flow CUDA aware, but I think I am going to bin that and just do a dense_optical_flow_cuda.py implementation.